.circleci/test/assembly/windows-zip/README.md

```diff
@@ -0,0 +1,61 @@
+# Workbase CI on Windows
```

Comment:
We need to put these code in `//test/assembly/...` rather than `.circleci/`, because `.circleci/` is hidden away from the main development environment and make it seem that this code is note "first class citizen" in the code base. And because we also want to treat `//test` as main code to take care of.

Reply:
Fixed in 1e1eb2e79b68e3d3952a045e5c6f9752ae37815d

The structure becomes like this:
```
test/
  assembly
  deployment
  common     # for things shared by both test-assembly and test-deployment (eg., 'GraknApplicationTest')
```